suite-native/module-connect-device/src/screens/ConnectingDeviceScreen.tsx

```diff
@@ -10,9 +10,10 @@ import { useTranslate } from '@suite-native/intl';
 
 import { ConnectDeviceBackground } from '../components/ConnectDeviceBackground';
 
-const screenStyle = prepareNativeStyle(() => ({
+const screenStyle = prepareNativeStyle(s => ({
     justifyContent: 'center',
     alignItems: 'center',
+    paddingBottom: 80,
```

Comment:
Is this to center the text? Should be done with flex to work on iPad as well

Reply:
Content is already centered, but I want it to be centered with bottom padding to move the text closer to the center, because it looks better on the screen (not it feels odd, because it is centered with the trezor logo on top of it)